css-fonts/styles.css

```diff
@@ -0,0 +1,8 @@
+body, h1, h2 {
+  font-family: 'Satisfy', cursive;
+  font-family: 'Merriweather', serif;
```

Comment:
This second `font-family` is overriding the first one, try moving it to your rule set for `p` elements.